modules/oe_content_publication/oe_content_publication.post_update.php

```diff
@@ -34,39 +33,72 @@ function oe_content_publication_post_update_00002() {
 }
 
 /**
- * Create Contact reference field and update form display.
+ * Install modules and clear cached plugin definitions.
  */
 function oe_content_publication_post_update_00003() {
   $module_installer = \Drupal::service('module_installer');
   if (!\Drupal::service('module_handler')->moduleExists('oe_content_organisation_reference')) {
     $module_installer->install(['oe_content_organisation_reference']);
   }
-  $module_installer->install(['path', 'inline_entity_form']);
-
-  $storage = new FileStorage(drupal_get_path('module', 'oe_content_publication') . '/config/post_updates/00003_add_contact_reference');
+  $module_installer->install([
```

Comment:
Lets be consistent here and check whether the modules are installed or not for all the modules we are going to install.

Reply:
ModuleInstaller::install() won't install already installed modules. It works faster then check modules separately. Might be   better for consistency to move  `oe_content_organisation_reference` to the list of installed modules?

Reply:
Good call.

Reply:
This is done.